src/selectors/rootedParentOf.ts

```diff
@@ -23,4 +23,8 @@ const rootedParentOf = <T extends Context | Path>(state: State, thoughts: T): T
       : (state.rootContext as T)
 }
 
+/** Calls rootedParentOf twice to get the rooted grandparent of a thought. */
+export const rootedGrandparentOf = <T extends Context | Path>(state: State, thoughts: T): T =>
+  rootedParentOf(state, rootedParentOf(state, thoughts))
```

Comment:
As a convention on this project, we generally only allow a single, default export that matches the file name. This makes it easier to know where functions live in the project, and is facilitated by the quick file picker that most editors provide. Reducers/Action Creators are an exception, and a few others, but we try to avoid this when possible.

Could move this into its own file in the `src/selectors` directory?